docs/source/deploying/prefix-path.md

```diff
@@ -0,0 +1,28 @@
+---
+myst:
+  html_meta:
+    "description": "Serve Volto on a subpath"
+    "property=og:description": "Serve Volto on a subpath"
+    "property=og:title": "Serve Volto on a subpath"
+    "keywords": "Volto, Plone, path, prefix, virtual hosting"
+---
+
+# Serve Volto on a subpath
+
+Volto can be served on a subpath, e.g. `https://mysite.com/subpath`.
+
+When this is enabled, both content and static assets are served on the subpath, which means a different website can be served at the root path.
```

Comment:
I think this is more explicit.

Can we serve multiple sites on multiple subpaths, or is it limited to just one?

```suggestion
When this is enabled, both content and static assets are served on the subpath.
This means different websites can be served at both the root path and the subpath.
```

Reply:
@stevepiercy I got caught on the wording of your second sentence (if they're different, then it's odd to use the word "both"). We're also making it sound like maybe both of these can be served through one Volto instance, but no, I'm saying it leaves the root path free in case you want to route it to different software via a frontend webserver.

How about:
```suggestion
When this is enabled, both content and static assets are served on the subpath.
This leaves the root path free in case you want to route it to a different website.
```